composer.json

```diff
@@ -10,10 +10,16 @@
     ],
     "require": {
         "php": "^7.1",
-        "phpspec/prophecy": "^1.7",
-        "phpunit/phpunit": "^6.0||^7.0",
         "phpstan/phpstan": "^0.10"
     },
+    "require-dev": {
+        "phpspec/prophecy": "^1.7",
+        "phpunit/phpunit": "^6.0||^7.0"
+    },
+    "conflicts": {
+        "phpspec/prophecy": "<1.7",
```

Comment:
Should we conflict with 2.0 too, to reflect the same behavior of requiring `^1.7`?